packages/react-reconciler/src/__tests__/ReactMemo-test.js

```diff
@@ -532,7 +532,7 @@ describe('memo', () => {
         ReactNoop.render(<MemoComponent optional="foo" />),
       ).toErrorDev(
         'Warning: Failed prop type: The prop `required` is marked as required in ' +
-          '`Foo`, but its value is `undefined`.\n' +
+          '`Bar`, but its value is `undefined`.\n' +
```

Comment:
This test was added in #18925 but I don't think this should've been the expected behavior @bvaughn.

For `React.memo` we ignore `displayName` on the wrapper if we have a `displayName` on the wrapped component.
For `React.forwardRef` we honor `displayName` on the wrapper even if we have `displayName` on the wrapped component.

I think it is confusing that `React.memo` and `React.forwardRef` behave differently with regard to `displayName`.